src/app/GitUI/UserControls/FolderBrowserButton.cs

```diff
@@ -18,6 +18,21 @@ public FolderBrowserButton()
         /// </summary>
         public Control? PathShowingControl { get; set; }
 
+        /// <summary>
+        /// Specifies the text label of the button.
+        /// </summary>
+        public override string Text
+        {
+            get
+            {
+                return buttonBrowse.Text;
+            }
+            set
+            {
+                buttonBrowse.Text = value;
+            }
+        }
```

Comment:
The following might reduce noise in the translation file:
```suggestion
        public override string _NO_TRANSLATE_Text
        {
            get => buttonBrowse.Text;
            set => buttonBrowse.Text = value;
        }
```

Reply:
My main point was and is to add the prefix `_NO_TRANSLATE_`.
Though it cannot be applied here, because it is an overridden property (what I did not notice in the first comment).
But it should work with renaming `buttonBrowse` (and setting the overridden property, too).
```suggestion
        public override string Text
        {
            get => base.Text;
            set
            {
                base.Text = value;
                _NO_TRANSLATE_buttonBrowse.Text = value;
            }
        }
```

Reply:
Does my last suggestion work?

Reply:
There is no need override `GitExtensionsControl.Text`.
```suggestion
        public string _NO_TRANSLATE_ButtonText
        {
            get => buttonBrowse.Text;
            set => buttonBrowse.Text = value;
        }
```
Maybe, this does not even need that prefix, i.e.
```suggestion
        public string ButtonText
        {
            get => buttonBrowse.Text;
            set => buttonBrowse.Text = value;
        }
```

Reply:
Your variant works well, too.